modules/standard/ChapelEnv.chpl

```diff
@@ -53,6 +57,7 @@ module ChapelEnv {
 
   /* Deprecated */
   pragma "last resort"
+  pragma "no doc"
```

Comment:
@mppf: While reviewing this, I came across this little "deprecated" description.  Should I just remove this variable even though it never really generated a user-facing deprecation message?  Or add a user-facing deprecation message now?  Looks like the only tests that refer to it are a bit meta:
* chplenv/chplconfig/correctness/correctness.chpl
* chplenv/printchplenv/printchplenv.chpl'
* compflags/albrecht/chplenv/chplenv.chpl

Reply:
Let's add a deprecation to it here and in ChplConfig.

Reply:
https://github.com/chapel-lang/chapel/pull/19227